Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerAnnotator.java

```diff
@@ -88,6 +74,27 @@ public AnnotatorInfo collectInformation(@NotNull final PsiFile psiFile, @Nullabl
     return new AnnotatorInfo(psiFile.getProject(), annotatedFile.getPath());
   }
 
+  public static boolean serverReadyForRequest(@NotNull PsiFile psiFile) {
+    if (psiFile instanceof DartExpressionCodeFragment) return false;
+
+    final Module module = ModuleUtilCore.findModuleForPsiElement(psiFile);
+    if (module == null) return true;
```

Comment:
Is 'return true' intentional here? To preserve logic that was before here should be 'return false'.